followthemoney/mapping/property.py

```diff
@@ -133,5 +135,18 @@ def map(
                 splote.extend(value.split(self.split))
             values = splote
 
+        discarded_values: List[str] = []
+
         for value in values:
-            proxy.unsafe_add(self.prop, value, fuzzy=self.fuzzy, format=self.format)
+            added_value = proxy.unsafe_add(
+                prop=self.prop,
+                value=value,
+                cleaned=(not self.clean),
```

Comment:
This does feel super dangerous to me. It means that no cleaning at all will be done: if the source data says "Russia", that'll be kept (instead of making `ru`), if the date is "Banana", all is considered good. Unless the input data is absolutely spotless, this will generate invalid FtM.

Reply:
Yes, you’re probably right. Rather than providing a generic escape hatch for cleaning it might be better to find solutions for concrete issues with validation. I’ll update the PR to drop this specific change, maybe having verbose logging whenever cleaning removes a value is enough for now…